extractors/other/aws-s3/index.md

```diff
@@ -88,18 +58,99 @@ You can add the following Policy Document as an Inline Policy to an AWS user:
 }
 {% endhighlight %}
 
-### Setting Up a New AWS S3 Bucket
+## Listing Tables
+
+{: .image-popup}
+![Screenshot - List tables](/extractors/other/aws-s3/ui9.png)
+
+The configuration can extract as many tables as you wish. 
+Each table has different settings (key, load type, etc.), but they all share the same credentials. 
+The list is fully searchable and you can delete or disable each table,
+or you can explicitly run extraction of only one table. Also the extraction order of the tables can be changed.  
+
+## Adding Tables
+
+{: .image-popup}
+![Screenshot - Create table](/extractors/other/aws-s3/ui3.png)
+
+To create a new table click the **New Table** button and assign a name. 
+This name will be used to create the destination table name in Storage and can be modified.
+ 
+
+### Specifying Files to Download
+
+{: .image-popup}
+![Screenshot - S3 Settings](/extractors/other/aws-s3/ui4.png)
+
+For each table you have to specify the AWS S3 bucket and a search key. 
+The search key can be a path to a single file or a prefix to multiple files 
+(omit the wildcard character and use the **Wildcard** checkbox instead).
+
+### Download Settings
+
+{: .image-popup}
+![Screenshot - Download Settings](/extractors/other/aws-s3/ui5.png)
+
+This section allows you to set up 
+
+ - **Wildcard** - S3 key is used as a prefix and all available files with the prefix will be downloaded
+ - **Subfolders** - Available only with **Wildcard** turned on. The extractor will also process all subfolders.
+ - **New Files Only** - The extractor will keep track of downloaded files and will continue with the unprocessed files 
+ on the next run.
+ - **Decompress** - All downloaded files will be decompressed (currently supporting ZIP and GZIP). 
+ Please note, ZIP files can contain multiple files, which can lead to ambiguity. We recommend using GZIP only.
```

Comment:
co se teda pak stane kdyz je to ambiguous

Reply:
sakra, seš moc pozornej, jsem myslel, že se tomuhle vyhnu. budu to muset vyzkoušet.

Reply:
budu muset napsat, že ZIP is not supported yet (nebo jen pro single file) - pokud bude víc zipů, tak podle https://github.com/keboola/processor-decompress#zip to vytvoří víc folderů a to pak nejde dál zpracovat. cesta z toho ven je dvojí, buď přidat do `processor-decompress` flag `flatten`, kterej udělá totéž jako `includeSubfolders` v https://github.com/keboola/aws-s3-extractor#configuration-options, nebo se udělá procesor `processor-flatten-folders`, kterej to vyřeší globálně i za extraktor.

Reply:
jdu to pořádně prozkoušet v různejch scénářích